src/Pages/2DPrinting/App.js

```diff
@@ -16,180 +11,128 @@ import {
-          className={this.props.className}
-        >
-          <ModalHeader toggle={this.toggle}>
-            Software & Computer Engineering Society
-          </ModalHeader>
-          <ModalBody>
-            <FilePond
-              ref={ref => (this.pond = ref)}
-              files={this.state.files}
-              allowMultiple
-              maxFiles={3}
-              maxFileSize='10MB'
-              server={this.serverOptions}
-              oninit={() => this.handleInit()}
-              onupdatefiles={fileItems => {
-                // Set currently active file objects to this.state
-                this.setState({
-                  files: fileItems.map(fileItem => fileItem.file)
-                })
-              }}
-            />
-
-            <br />
-
-            <FormGroup>
-              {/* <Label for="exampleSelect">Number of copies</Label> */}
-              <legend for='exampleSelect'>Number of copies</legend>
-              <Input type='select' name='select' id='exampleSelect'>
-                <option>1</option>
-                <option>2</option>
-                <option>3</option>
-                <option>4</option>
-                <option>5</option>
-              </Input>
-              <legend>Type of Print</legend>
-              <FormGroup check>
-                <Label check>
-                  <Input type='radio' name='radio1' /> Front
-                </Label>
-              </FormGroup>
-              <FormGroup check>
-                <Label check>
-                  <Input type='radio' name='radio1' />{' '}
-                  {/* add disabled if in need of option disablement */}
-                  Front & Back
-                </Label>
-              </FormGroup>
+export default function Printing (props) {
+  const [modal, setModal] = useState(false)
+  const [nestedModal, setNestedModal] = useState(false)
+  const files = []
+
+  return (
+    <div>
+      <Jumbotron>
+        <div className='text-center'>
+          <h1 className='display-4'>SCE Printing System</h1>{' '}
+        </div>
+      </Jumbotron>
+
+      <div className='printInfo'>
+        <p>
+          Welcome to printing! Click the icon below and upload your file. Each
+          member can print up to 30 pages a week.
+        </p>
+      </div>
+
+      {props.buttonLabel}
+      <FilePond
+        files={files}
+        allowMultiple={false}
+        acceptedFileTypes='application/pdf'
+        maxFileSize='10MB'
+        labelIdle="Drag & Drop or Touch Here <br />
+
+            <svg aria-hidden='true' viewBox='0 0 512 512' width='40%'>
+              <path
+                d='M399.95 160h-287.9C76.824 160 48 188.803 48
+              224v138.667h79.899V448H384.1v-85.333H464V224c0-35.197-28.825-64-64.05-64zM352
+              416H160V288h192v128zm32.101-352H127.899v80H384.1V64z'
+              />
+            </svg>"
+      />
+      <Button
+        color='primary'
+        className='continue'
+        // disabled={cont}
```

Comment:
remove this comment if you dont need it

Reply:
I pushed my stuff first using it right now. it's the hook for the continue button that starts out disabled